evaluate-itk-performance.py

```diff
@@ -114,6 +114,7 @@ def build_itk(itk_src, itk_bin):
         '-DCMAKE_CXX_STANDARD:STRING=11',
         '-DBUILD_TESTING:BOOL=OFF',
         '-DBUILD_EXAMPLES:BOOL=OFF',
+        '-DBUILD_SHARED_LIBS:BOOL=ON',
```

Comment:
I don't think this is required for TBB. I have been using TBB without this most of the time.

Reply:
Yes, I think that is correct. I added this in an attempt to address some odd CMake warnings, however, I think it is unrelated.

But, since this does not hurt, and it should improve link time for historical benchmark tests, I will leave it.